### ISSUE TITLE
boids not quite working

### ISSUE THREAD

**Issue by douglasmacdonald** [pyviz-topics/examples#143]
Hi,

Something does not look right with hosted boids demo.

https://boids.pyviz.demo.anaconda.com/notebooks/boids.ipynb

There are loads of warnings like

`WARNING:param.ParameterizedMetaclass: Use method 'params' via param namespace`

The result is that the final animation is not displaying.

**Comment by jbednar** [comment]
Thanks! The param warnings mean that we have pinned some packages too much and others not enough!  Most probably we pinned an old version of holoviews but did not also pin a matching old version of param. The warnings should not matter, though, so if there's an issue I don't think it would have to do with the warnings.

I tried visiting that link, and to me it does seem to be running. Yes, the many warnings are very annoying, but the final animation is there (which is _before_ the final code cell; the final cell starts it, but the animation is after the penultimate code cell).  I'm not sure it's working _properly_, though, as I only see three "boids"; I thought there should be dozens:

![image](https://user-images.githubusercontent.com/1695496/113225887-67b8bb00-9254-11eb-8f0f-306c0c9f038b.png)
